velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -405,30 +453,34 @@ std::optional<std::string> PrestoQueryRunner::toSql(
   for (auto i = 0; i < functions.size(); ++i) {
     appendComma(i, sql);
     sql << toCallSql(functions[i].functionCall);
```

Comment:
Just checking : toCallSql can handle IGNORE NULLs clause ?

Reply:
Hi @aditi-pandit, no, toCallSql in this PR doesn't handle IGNORE NULLS yet. This is because the current PR doesn't generate plans with IGNORE NULLS either. A subsequent PR (https://github.com/facebookincubator/velox/pull/8359) will make the fuzzer generate plans with IGNORE NULLS and handle it in PrestoQueryRunner.